### ISSUE TITLE
"Unsupported Windows version, will now exit."

### ISSUE THREAD

**Issue by DJuego** [tigros/Whisperer#40]
First. Thank you very much for your efforts in this project. I admit I haven't yet taken full advantage of what it has to offer but I love it! :-)

Unfortunately it has stopped working for me in its 3.0 version (2023-12-xx).  When I start it up I get the following error: "Unsupported Windows version, will now exit." And then the application terminates. 

I am using Windows 10 x64 fully upgraded (2023-12-xx). I have a NVidia GTX780 Ti card with updated drivers.

If you need more information just ask! Thanks in advance!

DJuego

**Comment by tigros** [comment]
something happened to your system, dunno what!

one thing to try from elevated cmd prompt: C:\Windows\SysWOW64\LODCTR /R

see if perfmon.exe works. try adding a counter do you see a "GPU Adapter Memory"?

**Comment by DJuego** [comment]
It works!

Initially the LODCTR /R call failed:

"Error: Unable to rebuild performance counter setting from system backup store, error code is 2"

 but I was able to run perfmon.exe without problems. And I added the counter also without problems. 

Then I ran the LODCTR /R call again and then...

"Info: Successfully rebuilt performance counter setting from system backup store"

I ran Whisperer and it worked this time! 

I don't know if the problem is definitely solved but anyway thank you very much for your support.

Merry Christmas! 

DJuego
